src/lib/constants.ts

```diff
@@ -33,14 +33,14 @@ export type FormatGroup = {
 
 export const FORMAT_GROUPS: FormatGroup[] = [
   {
-    name: 'sRGB Formats',
+    name: 'sRGB FORMATS',
```

Comment:
If we use CSS to transform the text to uppercase, that won't allow us to keep the s in sRGB lowercase. So I capitalized these at the source.